beetsplug/discogs.py

```diff
@@ -339,12 +341,21 @@ def get_album_info(self, result):
                          day=None, label=label, mediums=len(set(mediums)),
                          artist_sort=None, releasegroup_id=master_id,
                          catalognum=catalogno, script=None, language=None,
-                         country=country, albumstatus=None, media=media,
+                         country=country, style=style,
+                         albumstatus=None, media=media,
                          albumdisambig=None, artist_credit=None,
                          original_year=original_year, original_month=None,
                          original_day=None, data_source='Discogs',
                          data_url=data_url)
 
+    def format_style(self, style):
+        if style is None:
+            self._log.debug('Style not Found')
+        elif not style:
+            return style
```

Comment:
This seems to return `[]`, i.e., an empty list, instead of a string. I think it should be fine to remove this entire `elif` case: if the string is empty, then `'something'.join(sorted([]))` will return `''`, which is probably what we want.

Reply:
:D Think there was some confusion. When i was checking len == 0 it should have been len== 1. If there is only one style tag in the response then I want to just return that style tag as is with no formatting. So I guess I just needed to chage len == 0 to len == 1. Hope this clarifies what I am trying to achieve. If this makes sense I will make the change.

Reply:
It does! But I still don't think this `elif` case is necessary for that—a single style in the list will still "do nothing" when joined. For example:

```py
>>> ', '.join(['foo'])
'foo'
```

…so the one-element list case is also handled by the general case.

Reply:
👍 fixed